src/Database/Database.php

```diff
@@ -601,34 +601,60 @@ public function deleteAttribute(string $collection, string $id): bool
     }
 
     /**
-     * Delete Attribute
+     * Rename Attribute
      *
      * @param string $collection
-     * @param string $id
+     * @param string $old Current attribute ID
+     * @param string $name New attribute ID
      *
      * @return bool
      */
-    public function renameAttribute(string $collection, string $id, string $name): bool
+    public function renameAttribute(string $collection, string $old, string $new): bool
     {
         $collection = $this->getCollection($collection);
-
         $attributes = $collection->getAttribute('attributes', []);
+        $indexes = $collection->getAttribute('indexes', []);
+
+        $attribute = \in_array($old, \array_map(function($attribute) {
+            return $attribute['$id'];
+        }, $attributes));
```

Comment:
Just a minor thing, imo arrow functions are less noisy here.
```suggestion
        $attribute = \in_array($old, \array_map(fn($attribute) => $attribute['$id'], $attributes));
```